src/mflux_debugger/examples/tutorial_basic_pytorch.py

```
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
```

Ruff (0.14.3)

Shebang is present but file is not executable

(EXE001)